CMakeLists.txt

```diff
@@ -21,16 +21,10 @@
 PROJECT(mudlet)
 CMAKE_MINIMUM_REQUIRED(VERSION 3.1)
```

Comment:
How can we expect users to use the latest version of our software when we can't use the latest ourselves - that's my line of thinking...